cli.py

```diff
@@ -68,18 +69,26 @@ def main(log_level):
               type=str, help="D3A domain name")
 @click.option('-w', '--web-socket', default=DEFAULT_WEBSOCKET_DOMAIN,
               type=str, help="D3A websocket URL")
+@click.option('-j', '--json-file-name', type=str,
+              help="JSON Filename containing info related to simulation")
 @click.option('--run-on-redis', is_flag=True, default=False, help="Start the client using the Redis API")
-def run(setup_module_name, username, password, domain_name, web_socket, run_on_redis, **kwargs):
+def run(setup_module_name, username, password, domain_name, web_socket, json_file_name,
+        run_on_redis, **kwargs):
     if username is not None:
         os.environ["API_CLIENT_USERNAME"] = username
     if password is not None:
         os.environ["API_CLIENT_PASSWORD"] = password
     os.environ["API_CLIENT_DOMAIN_NAME"] = domain_name
     os.environ["API_CLIENT_WEBSOCKET_DOMAIN_NAME"] = web_socket
     os.environ["API_CLIENT_RUN_ON_REDIS"] = "true" if run_on_redis else "false"
+    os.environ["JSON_FILE_PATH"] = os.path.join(modules_path[0], json_file_name)
 
     try:
-        importlib.import_module(f"d3a_api_client.setups.{setup_module_name}")
+        if SETUP_FILE_PATH:
+            sys.path.append(SETUP_FILE_PATH)
+            importlib.import_module(f"{setup_module_name}")
+        else:
+            importlib.import_module(f"d3a_api_client.setups.{setup_module_name}")
```

Comment:
I dont see the SETUP_FILE_PATH set somewhere neither a way the user could set it. Can you help me please?

Reply:
For now it could be set via user and found in [constants](https://github.com/gridsingularity/d3a-api-client/blob/master/d3a_api_client/constants.py#L7)

Reply:
> I dont see the SETUP_FILE_PATH set somewhere neither a way the user could set it. Can you help me please?

In situations like this I like to use namespace imports rather than direct imports:
```python
# Direct import
from d3a_api_client.constants import SETUP_FILE_PATH

# Namespace import
from d3a_api_client import constants
```

So the call would become `constants.SETUP_FILE_PATH`, which makes it immediately obvious that the object comes from a different module (and that it is a constant, given the name of that module).

Reply:
The purpose of having a CLI tool in my perspective is to relief user for the need to go inside `api-client` library code and  had to overwrite any constant settings.  I will add a flag for api-client to read client's scripts other than the default setup path.